model-api/src/jsMain/kotlin/org/modelix/model/api/NodeAdapterJS.kt

```diff
@@ -97,6 +98,10 @@ class NodeAdapterJS(val node: INode) : INodeJS_ {
         node.remove()
     }
 
+    override fun wrap(): ITypedNode {
```

Comment:
Consider making the code more readable by giving the existing method a short alias in you project.

```
import { LanguageRegistry } from "@modelix/ts-model-api"
const wrap = LanguageRegistry.INSTANCE.wrapNode
// maybe const wrap = LanguageRegistry.INSTANCE.wrapNode.bind(LanguageRegistry.INSTANCE)

const untypedNode: INodeJS = foo();
const typedNode = wrap(untypedNode);

// instead of
import { LanguageRegistry } from "@modelix/ts-model-api";
const untypedNode: INodeJS = foo();
const typedNode = LanguageRegistry.INSTANCE.wrapNode(untypedNode);
```

Reply:
More usefull for you could be to generate methods like `function wrap_PrimitiveTypeRef(node: INode): PrimitiveTypeRef` similar to ` function isOfConcept_PrimitiveTypeRef(node: ITypedNode): node is PrimitiveTypeRef`.